articles/dev-box/concept-dev-box-concepts.md

```diff
@@ -21,12 +21,10 @@ This article describes the key concepts and components of Microsoft Dev Box.
 
 ## Dev center
 
-A dev center is a collection of projects that require similar settings. Dev centers enable dev infrastructure managers to manage the images and SKUs available to the projects using [dev box definitions](concept-dev-box-concepts.md#dev-box-definition), and configure the networks the development teams consume using [network connections](./concept-dev-box-concepts.md#network-connection).  
-
+A dev center is a collection of projects that require similar settings. Dev centers enable dev infrastructure managers to manage the images and SKUs available to the projects using [dev box ](concept-dev-box-concepts.md#dev-box-definition)definitions and configure the networks the development teams consume using [network connections](./concept-dev-box-concepts.md#network-connection).  
```

Comment:
Did you remove the word "definitions" from the link text? It should be included.

Reply:
apologies in the delay.  I have added the "definitions" back